### PR TITLE
[CRT]  Rename unimplemented mutation tree

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Eliminates #53095 from discussion.

#### Describe the solution
Wendigo is an unimplemented mutation tree whose mutagen can only be gained via debug.   Therefore none of these changes should need a migration.  @ZhilkinSerg please confirm that I'm not going to be breaking saves by this.

#### Describe alternatives you've considered
none

### CODE REVIEW

**Comment by andrei8l** [comment]
If you're giving in to a bigot to avoid discussion, doesn't that make you a bigot as well? Have you at least discussed this with CRT's maintainer?

**Comment by Zireael07** [comment]
@andrei8l: I fail to see where the person who made the issue is a bigot - on the contrary, the original name of the tree is indeed a taboo in Native American cultures.

**Comment by Maddremor** [comment]
As the resident CRIT crank, is there a reason not to just remove it, or are you working with a cohesive theme in mind?

**Comment by andrei8l** [comment]
> I fail to see where the person who made the issue is a bigot - on the contrary, the original name of the tree is indeed a taboo in Native American cultures.

He seeks to impose the restrictions of one culture on the rest of us while dismissing identical issues from other cultures. AFAIK, CDDA does not cater to any specific group so the fairest approach is to ignore all such restrictions and leave them up to the player. We're getting off topic for this PR though so let's move this issue somewhere else.

**Comment by Fris0uman** [comment]
If you're sure that the mutation can't be optained in game, then changing the id will only break saves where someone used debug to get them.

Also we really need to do something about CRIT, the thing still has no coherent theme ( as far as I know) and apparently has whole mutation tree that do nothing.

**Comment by Rivet-the-Zombie** [comment]
I really do not want to set a precedent of removing content of any sort due to religious and/or other superstitious people claiming offense to our usage of a word. If it's a word they don't want to use themselves, so be it. What they shouldn't be doing is telling other people who don't share their particular beliefs 'that's our word and you're not allowed to use it'.

**Comment by Maleclypse** [comment]
> If you're giving in to a bigot to avoid discussion, doesn't that make you a bigot as well? Have you at least discussed this with CRT's maintainer?

For around a year I have been the only person maintaining CRT. And no I don’t believe in s transitive property of bigotry.

**Comment by Maleclypse** [comment]
> I really do not want to set a precedent of removing content of any sort due to religious and/or other superstitious people claiming offense to our usage of a word. If it's a word they don't want to use themselves, so be it. What they shouldn't be doing is telling other people who don't share their particular beliefs 'that's our word and you're not allowed to use it'.

As the only person currently working on CRT I looked at the work already done on this non-working mutation tree.   I looked at the amount of work I would need to do in order for it to appear in game.  I decided that the current tree has the name of a cannibal monster and the mutations of a natures avenger and that if I wanted to work on it I wanted to push it one direction or the other.  Not some weird mishmash, so telling me I can't change the name is telling me I can't work on it and if you're telling me I can't work on it because I'm editing original authors intent then I'm going to stop working on CRT and with no active maintainer CRT will go obsolete.

**Comment by Photoloss** [comment]
Out of curiosity how would you differentiate the "Sylvan" approach from Elf-A and Plant, especially if the latter gets split into "Treant" and "Flower" lines?

Mind you I could ask the same for cannibal-Wendigo and Beast, Chimera or a line based on the skeletal zombies. The current Wendigo looks very much like a typical "half and half" mythological mishmash, if Elfa is Plant+Alpha then Wendigo is Plant+Beast.

**Comment by Maleclypse** [comment]
> Out of curiosity how would you differentiate the "Sylvan" approach from Elf-A and Plant, especially if the latter gets split into "Treant" and "Flower" lines?
> 
> Mind you I could ask the same for cannibal-Wendigo and Beast, Chimera or a line based on the skeletal zombies. The current Wendigo looks very much like a typical "half and half" mythological mishmash, if Elfa is Plant+Alpha then Wendigo is Plant+Beast.

That's part of the problem, I'm very much of the mind that all mutation trees should have a very specific play style and that the mutation descriptions should be a cosmetic reflection of that.   The unique mutations this path has already, gave me a feeling of "ivy but tentacles"  so I've got to look at how I make this something different from "plant but better"  (better in this case means more combat focused). Like you've stated a possibility is to make this plant+beast so it's an exotic/advanced tree like raptor.  I'd like to make it more unique like medical as far as having few crossover abilities.  Which means more work.

**Comment by LeahLuong** [comment]
Mal, I saw your & Kev's final comments in the other thread & they seemed @ odds but here you are seemingly taking Kev's lead on this. Why is this?? If you are the only 1 working on the mod these days, isn't the decision to change the name yours to make??

You mentioned you don't know the original author's intent. Do you know their cultural heritage?? No 1 in the previous thread proved the original author WASN'T of Algonquian descent. Nor was the taboo against speaking of the wendigo demonstrated to hold across the many tribes who reference that meme. The best I've found in my searches are 2nd-hand accounts of some1 claiming they know some1 who says it's taboo to say its name. I think the name fits the concept you are working w/ quite well. & I can only echo the sentiment of others in this thread & the previous 1 that by acceding to the demand for name change you are setting a precedent that may reverberate through many other parts of the game as well, not just w/in this single mod.

**Comment by Maleclypse** [comment]
> Mal, I saw your & Kev's final comments in the other thread & they seemed @ odds but here you are seemingly taking Kev's lead on this. Why is this?? If you are the only 1 working on the mod these days, isn't the decision to change the name yours to make??
> 
1. You seem to be confused on this. Kevin 100% said that taboo claims would not impact Cataclysm.  Kevin is fine with the name staying Wendigo. 
> You mentioned you don't know the original author's intent. Do you know their cultural heritage?? No 1 in the previous thread proved the original author WASN'T of Algonquian descent. Nor was the taboo against speaking of the wendigo demonstrated to hold across the many tribes who reference that meme. The best I've found in my searches are 2nd-hand accounts of some1 claiming they know some1 who says it's taboo to say its name. I think the name fits the concept you are working w/ quite well. & I can only echo the sentiment of others in this thread & the previous 1 that by acceding to the demand for name change you are setting a precedent that may reverberate through many other parts of the game as well, not just w/in this single mod.

2. I know the original author and they spoke about their family enough that I am 100% certain they are not Algonquin unless they were actually were and were pretending not to be Algonquin just to screw with people.  If that far fetched scenario were in fact true then screw them for lying.  

3. As referenced above I'm not changing the name due to any taboo but specifically because I have a different vision for what the mutations would look like for a Wendigo path than what is already there.

So in conclusion no precedent is being and even if I were trying to set a precedent or someone else took it as such, Kevin has full on said he will not consider these types of requests.

**Comment by kevingranade** [comment]
For example, in a similar vein we had a demand for how we used the term Vacutainer that was addressed in #21486
The claim itself was baseless, as I consider the one that triggered this one to be, but in a similar way, once we looked at usage of the term, it was inaccurate, so we changed it.

**Comment by LeahLuong** [comment]
Kevin closed last thread by saying:

> Closing, @Maleclypse may find the secondary inaccuracy argument compelling, but the primary argument of avoiding it for cultural appropriation or taboo is not.

I took "it" to mean "name change" although looking @ the phrasing again, the statement reads ambiguously to me. I get the feeling you 2 were having a convo on this subject outside the threads & I'm prob lacking some context. Anyway, thanks for clearing that up for me.

This whole thing was kind of an exercise of arguing for the sake of argument to me. I play vanilla (for now) so I don't really care much what happens in the mods but the concept of "cultural appropriation" is something that really gets my goat. Had Doggo taken the tack of wendigo being a supernatural, spiritual entity unlikely to leave a genetic legacy subject to emulation through mutagens, I'd prob not have joined the fray. It's good to hear you guys are mostly in agreement on this w/ me. Participants' emoji use across both threads seem to reflect this too.

**Comment by NoahGoldFox** [comment]
This needs to be reversed, the internet has adopted wendigos and improved them, they arent just a indian thing anymore. Modern internet-style wendigos are *way* better than the boring original lore indians made.

**Comment by Maddremor** [comment]
> This needs to be reversed, the internet has adopted wendigos and improved them, they arent just a indian thing anymore. Modern internet-style wendigos are _way_ better than the boring original lore indians made.

WTF is this take? There can be a nuanced discussion of cultural appropriation and what is appropriation, sharing, or cultural cross pollination, but this is not that. What made you think insulting a marginalized group like this was a good idea?

**Comment by kevingranade** [comment]
NoahGoldFox has been a persistent source of noise in the repo, and this is the last straw.
